game-core/src/main/java/games/strategy/triplea/oddsCalculator/ta/BattleResults.java

```diff
@@ -9,13 +9,17 @@
 import games.strategy.triplea.delegate.IBattle.WhoWon;
 import games.strategy.triplea.delegate.Matches;
 import games.strategy.util.CollectionUtils;
+import lombok.Getter;
 
 public class BattleResults extends GameDataComponent {
```

Comment:
I'm pretty sure `BattleResults` _is_ serialized with game data.  Renaming these fields is probably going to break compatibility.

Reply:
You are right, and it did break compat. It seemed it was more the re-order, out of order was still broken. We may be able to change the names; but even if we can, explicitly declaring the methods seems better. This is fixed in: 1696563

After that update I was not able to get any breakages with old+new client in battle or tech cases. To repro the breakage took a good bit, had to get all the way to a networked battle.